addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -93,6 +95,8 @@ String myKodi_input         "Input"                 { channel="kodi:kodi:myKodi:
 String myKodi_inputtext     "Inputtext"             { channel="kodi:kodi:myKodi:inputtext" }
 String myKodi_systemcommand "Systemcommand"         { channel="kodi:kodi:myKodi:systemcommand" }
 String myKodi_mediatype     "Mediatype [%s]"        { channel="kodi:kodi:myKodi:mediatype" }
+String myKodi_thumbnail                             { channel="kodi:kodi:myKodi:thumbnail" }
+String myKodi_fanart                                { channel="kodi:kodi:myKodi:fanart" }
```

Comment:
I added an example but I could not test whether it works because I currently have no access to a running Kodi. In fact I would prefer to use in in HABPanel instead.